tests/integration/tests/traces_spans_test.go

```diff
@@ -104,14 +104,20 @@ func assertTraces(traces *model.TracingResponse, statusCode int, err error, requ
 }
 
 func assertSpans(spans []model.TracingSpan, statusCode int, err error, require *require.Assertions) {
+	tracing, _, errTracing := kiali.TracingConfig()
+	require.NoError(errTracing)
+
 	if statusCode == 200 {
 		require.NoError(err)
 		require.NotNil(spans)
 		if len(spans) > 0 {
 			require.NotNil(spans[0].TraceID)
-			require.NotEmpty(spans[0].References)
 			require.NotNil(spans[0].References[0].TraceID)
 			require.Equal(spans[0].TraceID, spans[0].References[0].TraceID)
```

Comment:
shouldn't these two be moved too? (since References array is empty)

Reply:
Ahh good catch, @mkralik3 ! I've just tested with Jaeger. Tested with Tempo now.

Reply:
Thanks :) I am going to try it with OSSM3 downstream pipeline